tests/acceptance/let-helper-test.js

```diff
@@ -94,6 +94,22 @@ describe('Acceptance: let helper', function() {
         });
       });
 
+      it('scopes to outmost scope', () => {
+        andThen(() => {
```

Comment:
It doesn't seem to work. I think it's related to the fact that we're writing acceptance tests which use `visit` & `click` async helpers.